lib/ansible/modules/pause.py

```diff
@@ -36,6 +36,12 @@
     type: bool
     default: 'yes'
     version_added: 2.5
+  silent:
+    description:
+      - Do not show info output before pausing
```

Comment:
I really like the idea of cross linking the `pause`/`wait_for` docs so that's my preference. But I do have a suggestion if this were to be merged.

This should be specific. For example, take a look at the echo option description.
```suggestion
      - Controls whether or not the duration and how to interrupt the timeout it is shown.
      - Has no effect unless 'seconds' or 'minutes' is set.
      - Has no effect on the final message if the timer is interrupted with ctrl+C.
```

And it would need a [changelog](https://docs.ansible.com/ansible/latest/community/development_process.html#creating-changelog-fragments) and [tests](https://github.com/ansible/ansible/tree/devel/test/integration/targets/pause) like all new features.